.github/workflows/build.yaml

```diff
@@ -19,16 +19,16 @@ jobs:
           - target: x86_64-unknown-linux-gnu
             args: "--features full"
 
-          # For `musl` builds openssl must be vendored
+          # For `musl` builds, use the `static` feature.
```

Comment:
Please adjust this comment mentioning the `static` feature.

There are multiple occurrences here in `build.yaml` and `quality.yaml`

Reply:
Done